registration-processor/workflow-engine/registration-processor-workflow-manager-service/pom.xml

```diff
@@ -33,6 +33,11 @@
 			<artifactId>vertx-core</artifactId>
 			<version>${vertx.version}</version>
 		</dependency>
+		<dependency>

```

Comment:
auth adapter cant be put as a maven dependency. It should be passed in docker during runtime. Please check all the spring boot services in mosip to get example.
Dockerfile should be changed.